sunpycube/cube/tests/test_NDCube.py

```diff
@@ -0,0 +1,208 @@
+# -*- coding: utf-8 -*-
+'''
+Tests for NDCube
+'''
+from __future__ import absolute_import
```

Comment:
Is this import needed?  I thought importing from future was about getting Python 3 functionality into Python 2.  And if we aren't supporting Python 2 that's unnecessary.  I could be wrong about importing from future though...